src/govuk/components/cookie-banner/_index.scss

```diff
@@ -1,10 +1,55 @@
 @include govuk-exports("govuk/component/cookie-banner") {
-  // For supporting older browsers which don't hide elements with the `hidden` attribute
+
+  // This needs to be kept in sync with the header component's styles
+  $border-bottom-width: govuk-spacing(2);
+
+  .govuk-cookie-banner {
+    @include govuk-font($size: 19);
+
+    padding-top: govuk-spacing(4);
+
+    // The component does not set bottom spacing.
+    // The bottom spacing should be created by the items inside the component.
+    // Clear out internal bottoms margins on things, for instance those set by .govuk-button-group,
+    // so that they collapse instead of "leaking" outside the container.
+    @include govuk-clearfix;
```

Comment:
I feel like I'm probably missing something here, so sorry if I am! But is this still needed? If I remove the clearfix, it doesn't seem to break? I can also see the grid class has its own clearfix so just wondering if we do need it, do we need both?

Reply:
@vanitabarrett Good catch. It's actually the new margin for the transparent border that's now "pulling in" the bottom margin. I'll remove this 👍